cloud/features/verified-commits.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Verified Commits
+description: Learn about verifying Git commits with Flipt Cloud
+mode: "wide"
+---
+
+Verify commits to your GitHub repository with SSH signing keys.
```

Comment:
maybe we could add a short section on why you would want to verify commits? perhaps linking to the GitHub docs on verified commits too?